CHANGELOG.md

```diff
@@ -227,9 +231,7 @@
 
 -   Implement basic support for Http connector for Flink SQL
 
-[Unreleased]: https://github.com/getindata/flink-http-connector/compare/0.19.0...HEAD
-
-[0.19.0]: https://github.com/getindata/flink-http-connector/compare/0.18.0...0.19.0
+[Unreleased]: https://github.com/getindata/flink-http-connector/compare/0.18.0...HEAD
```

Comment:
Why this change?

If I recall correctly the change log is "bumped" automatically when releasing new version.

The only thing needed during the development is to add a new entry under [undrelesed] section

Reply:
I guess this is due to incorrect rebase with main. This branch was created before I merged PR "Bump development version after release of 0.19.0"